### ISSUE TITLE
Multiplayer Explained Issue on Alpha 12

### ISSUE THREAD

**Issue by ChildLearningClub** [MitchMakesThings/Godot-Things#4]
Hi Mitch, I'm attempting to run through the networking tutorial that you created and having trouble with running the .bat files in windows. I modified it accordingly but I'm getting these errors when running it.

```
"C:\Program Files\Godot\Godot_v4.0-alpha12_win64.exe" --path .\Multiplayer\ --server --headless

Godot Engine v4.0.alpha12.official.2c11e6d9e - https://godotengine.org

WARNING: res://Characters/Aliens/Player.tscn:3 - ext_resource, invalid UUID: uid://yro4ypo6sek5 - using text path instead: res://Characters/Aliens/Blue/BlueSpriteFrames.tres
     at: load (scene/resources/resource_format_text.cpp:440)
WARNING: res://Characters/Aliens/Blue/BlueSpriteFrames.tres:3 - ext_resource, invalid UUID: uid://xjnyqosqdmw1 - using text path instead: res://Characters/Aliens/Blue/alienBlue_walk1.png
     at: load (scene/resources/resource_format_text.cpp:440)
ERROR: Unable to open file: res://.godot/imported/alienBlue_walk1.png-977551bdfe3f92f98b7269f8ed7e31ba.ctex.
   at: (scene/resources/texture.cpp:822)
ERROR: Failed loading resource: res://.godot/imported/alienBlue_walk1.png-977551bdfe3f92f98b7269f8ed7e31ba.ctex. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: Failed loading resource: res://Characters/Aliens/Blue/alienBlue_walk1.png. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: res://Characters/Aliens/Blue/BlueSpriteFrames.tres:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://Characters/Aliens/Blue/alienBlue_walk1.png
   at: load (scene/resources/resource_format_text.cpp:478)
ERROR: Failed loading resource: res://Characters/Aliens/Blue/BlueSpriteFrames.tres. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: res://Characters/Aliens/Player.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://Characters/Aliens/Blue/BlueSpriteFrames.tres
   at: load (scene/resources/resource_format_text.cpp:478)
ERROR: Failed loading resource: res://Characters/Aliens/Player.tscn. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: Parser bug (please report): tried to assign unset node without an identifier.
   at: resolve_class_interface (modules/gdscript/gdscript_analyzer.cpp:656)
SCRIPT ERROR: Parse Error: Could not preload resource file "res://Characters/Aliens/Player.tscn".
          at: GDScript::reload (res://System/main.gd:4)
SCRIPT ERROR: Parse Error: Cannot use simple "@export" annotation because the type of the initialized value can't be inferred.
          at: GDScript::reload (res://System/main.gd:3)
ERROR: Method/function failed. Returning: ERR_PARSE_ERROR
   at: reload (modules/gdscript/gdscript.cpp:886)
WARNING: res://System/main.tscn:4 - ext_resource, invalid UUID: uid://dhvi1o1ili7f - using text path instead: res://Maps/TestMap.tscn
     at: load (scene/resources/resource_format_text.cpp:440)
WARNING: res://Maps/TestMap.tscn:3 - ext_resource, invalid UUID: uid://bp0gb6ml2ruad - using text path instead: res://Maps/Tileset.tres
     at: load (scene/resources/resource_format_text.cpp:440)
WARNING: res://Maps/Tileset.tres:3 - ext_resource, invalid UUID: uid://cday0n3k0g4ew - using text path instead: res://Maps/tiles_spritesheet.png
     at: load (scene/resources/resource_format_text.cpp:440)
ERROR: Unable to open file: res://.godot/imported/tiles_spritesheet.png-9be423446f4a81a250967ebe388bb064.ctex.
   at: (scene/resources/texture.cpp:822)
ERROR: Failed loading resource: res://.godot/imported/tiles_spritesheet.png-9be423446f4a81a250967ebe388bb064.ctex. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: Failed loading resource: res://Maps/tiles_spritesheet.png. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: res://Maps/Tileset.tres:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://Maps/tiles_spritesheet.png
   at: load (scene/resources/resource_format_text.cpp:478)
ERROR: Failed loading resource: res://Maps/Tileset.tres. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: res://Maps/TestMap.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://Maps/Tileset.tres
   at: load (scene/resources/resource_format_text.cpp:478)
ERROR: Failed loading resource: res://Maps/TestMap.tscn. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: res://System/main.tscn:4 - Parse Error: [ext_resource] referenced nonexistent resource at: res://Maps/TestMap.tscn
   at: load (scene/resources/resource_format_text.cpp:478)
ERROR: Failed loading resource: res://System/main.tscn. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
ERROR: Failed loading scene: res://System/main.tscn
   at: (main/main.cpp:2669)
ERROR: Condition "_first != nullptr" is true.
   at: ~List (./core/templates/self_list.h:106)
ERROR: Condition "_first != nullptr" is true.
   at: ~List (./core/templates/self_list.h:106)
ERROR: 1 RID allocations of type 'P11GodotBody3D' were leaked at exit.
ERROR: 1 RID allocations of type 'P11GodotArea3D' were leaked at exit.
ERROR: 1 RID allocations of type 'P12GodotSpace3D' were leaked at exit.
ERROR: 1 RID allocations of type 'P11GodotArea2D' were leaked at exit.
ERROR: 1 RID allocations of type 'P12GodotSpace2D' were leaked at exit.
ERROR: 2 RID allocations of type '6NavMap' were leaked at exit.
ERROR: 1 RID allocations of type 'N18RendererCanvasCull6CanvasE' were leaked at exit.
ERROR: 1 RID allocations of type 'N16RendererViewport8ViewportE' were leaked at exit.
ERROR: 1 RID allocations of type 'PN13RendererDummy14TextureStorage12DummyTextureE' were leaked at exit.
ERROR: 1 RID allocations of type 'N17RendererSceneCull8ScenarioE' were leaked at exit.
WARNING: ObjectDB instances leaked at exit (run with --verbose for details).
     at: cleanup (core/object/object.cpp:1918)
ERROR: Resources still in use at exit (run with --verbose for details).
   at: clear (core/io/resource.cpp:474)


```
I assume it is something that I'm doing incorrectly, but wanted to make sure?

**Comment by MitchMakesThings** [comment]
I've just tried checking out and running it on another computer and it seems to be working fine for me.
Are you sure you've opened the project with the editor at least once before trying to run it with the batch file? - I'm not sure if it's necessary to run the project from the editor once first, but that's something I did.

Failing that, you might need to delete all the *.import files, then open the editor again to force it to reimport everything.

**Comment by ChildLearningClub** [comment]
Thank you Mitch. I tried your solution which lead me to my issue. I had copied "Multiplayer" folder out of my downloads folder, but not the .bat files :(. So I was running the .bat files from the downloads folder which where refencing the "Multiplayer" games files in the downloads folder as you and the error indicated.. I apologize for my mistake. Thank you for helping me resolve it and for your tutorial. I could not get multiplayer working without you honestly. So I greatly appreciate your tutorials and help.
